src/ui/components/Events/Events.tsx

```diff
@@ -30,20 +45,18 @@ function Events({ currentTime, events, executionPoint, seek }: PropsFromRedux) {
   );
 
   return (
-
-    <div className="flex flex-col py-1.5 self-stretch space-y-1.5 w-full text-xs bg-white">
-    
+    <div className="flex flex-col py-1.5 self-stretch w-full text-xs bg-white">
```

Comment:
I think the only classes that are doing anything here are: `py-1.5 text-xs bg-white`?

Reply:
👍